app/src/main/java/com/duckduckgo/app/feedback/db/SurveyDao.kt

```diff
@@ -41,11 +44,21 @@ abstract class SurveyDao {
     @Query("""select * from survey where status = "SCHEDULED"""")
     abstract fun getScheduled(): List<Survey>
 
+    @Query("""select * from survey where status = "SCHEDULED" or status = "NOT_ALLOCATED"""")
+    abstract fun getUnused(): List<Survey>
+
     @Transaction
     open fun cancelScheduledSurveys() {
         getScheduled().forEach {
             it.status = Survey.Status.CANCELLED
             update(it)
         }
     }
+
+    @Transaction
+    open fun deleteUnusedSurveys() {
+        getUnused().forEach {
```

Comment:
Since we only use the `getUnused()` query for the purposes of deleting, can we do that directly in an SQL statement? i.e.,

    @Query("""delete from survey where status = "SCHEDULED" or status = "NOT_ALLOCATED"""")
    abstract fun deleteUnusedSurveys()

Reply:
Great suggestion, thanks